src/pages/SubTopic.tsx

```diff
@@ -123,26 +138,37 @@ const Topic = () => {
             </BreadCrumbs>
           </Typography>
           <hr />
-          <div className="pdf-container">
+          <div
+            className="pdf-container"
+            ref={containerRef}
+            style={{ overflow: 'scroll', height: '100vh' }}>
```

Comment:
I prefer switch this to Box and use sx. Using style directly is a bad practice. You could create a styled componetn as wel
```suggestion
          <Box
            className="pdf-container"
            ref={containerRef}
            overflow="scroll"
            height="100vh">
```